### ISSUE TITLE
OMD Pipelines fails to discover Airflow if AIRFLOW__WEBSERVER__BASE_URL defined

### ISSUE THREAD

**Issue by Sudokamikaze** [open-metadata/OpenMetadata#15251]
**Affected module**
Unable to work with OMD as it can't connect to Airflow

**Describe the bug**
When Airflow has defined custom base_url (AIRFLOW__WEBSERVER__BASE_URL) to respond on different URL path (i.e http://localhost/airflow) OMD fails to connect to it even if `PIPELINE_SERVICE_CLIENT_ENDPOINT` has correct address inside

To debug we were able to exec
```
curl -XPOST http://localhost:8088/airflow/api/v1/openmetadata/enable --data-raw '{"dag_id": "DAG_NAME"}' -u "admin:admin" --header 'Content-Type: application/json'
```

As you may see the API path is `/airflow/api/v1/openmetadata/enable` not `/api/v1/openmetadata/enable`

Prior 1.3.0 (1.2.5 in our case) this configuration worked, right now it doesn't, to me it looks like OMD doesn't respect additional data passed to `PIPELINE_SERVICE_CLIENT_ENDPOINT` and only takes FQDN & port

**To Reproduce**
* Get https://github.com/open-metadata/OpenMetadata/blob/1.3.0/docker/docker-compose-quickstart/docker-compose.yml
* Add `AIRFLOW__WEBSERVER__BASE_URL: "http://localhost:8080/airflow"` [here](https://github.com/open-metadata/OpenMetadata/blob/1.3.0/docker/docker-compose-quickstart/docker-compose.yml#L448)
* Change to `PIPELINE_SERVICE_CLIENT_ENDPOINT: http://ingestion:8080/airflow  [here](https://github.com/open-metadata/OpenMetadata/blob/1.3.0/docker/docker-compose-quickstart/docker-compose.yml#L140) and [here](https://github.com/open-metadata/OpenMetadata/blob/1.3.0/docker/docker-compose-quickstart/docker-compose.yml#L325)
* Start compose, go to settings, pipelines, you'll see it will fail with an error "Please install managed APIs"

**Expected behavior**
Working pipelines UI and its configuration

**Version:**
 - OpenMetadata version: 1.3.0
 - OpenMetadata Ingestion package version: official OMD ingestion docker image 1.3.0

**Additional context**

**Comment by khalepski** [comment]
any ETA when you are going to take care of it?

**Comment by pmbrull** [comment]
will be released next week on 1.3.1
